apps/desktop/package.json

```diff
@@ -45,6 +45,7 @@
     "@cypherock/tsconfig": "workspace:^",
     "@electron/notarize": "^1.2.3",
     "@jest/globals": "^29.4.1",
+    "@playwright/test": "^1.34.3",
```

Comment:
Add a script in package.json to run `e2e` tests

Reply:
e2e has been added in script